projects/storefrontlib/src/lib/global-message/components/global-message.component.spec.ts

```diff
@@ -1,63 +1,56 @@
 import { ComponentFixture, TestBed, async } from '@angular/core/testing';
-import { Store, StoreModule } from '@ngrx/store';
 import { of } from 'rxjs';
+import createSpy = jasmine.createSpy;
 import { GlobalMessageComponent } from './global-messsage.component';
-import * as fromGlobalMessage from '../store';
 import { GlobalMessageType } from './../models/message.model';
-
+import { GlobalMessageService } from '../facade/global-message.service';
+
+const mockMessages = new Map<GlobalMessageType, string[]>();
+mockMessages.set(GlobalMessageType.MSG_TYPE_CONFIRMATION, ['Confirmation']);
+mockMessages.set(GlobalMessageType.MSG_TYPE_CONFIRMATION, ['Confirmation']);
```

Comment:
There seems to be a line duplication:
```
mockMessages.set(GlobalMessageType.MSG_TYPE_CONFIRMATION, ['Confirmation']);
mockMessages.set(GlobalMessageType.MSG_TYPE_CONFIRMATION, ['Confirmation']);
```

Reply:
This is not duplication. This one test that: if we add duplicate messages, we only keep one in the store. So, the number of confirmation message is 1.

Reply:
Got it.  Thanks for the explanation!